plugins/operator_plugin/src/services/cron_service.cpp

```diff
@@ -96,8 +96,14 @@ namespace sysio::operator_plugin::services {
     if (j) {
       j->cancelled = true;
       if (j->timer) {
-        boost::system::error_code ec;
-        j->timer->cancel(ec);
+        try {
+          j->timer->cancel();
+        } catch (const std::exception& e) {
+          logger().log(FC_LOG_MESSAGE(warn,
+            "cron_service::cancel() timer->cancel() threw: ${what}",
+            ("what", e.what())
+          ));
```

Comment:
Should just be: `fc_wlog(logger(), "cron_service::cancel() timer->cancel() threw: ${what}", ("what", e.what());`